foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -158,6 +158,16 @@ Writing an HTML comment is simple: We just enclose the comment with `<!--` and `
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
+<div class="lesson-note" markdown="1">
+
+#### A note about HTML Comments
+
+Let's be honest, constantly typing `<!--` and `-->` tags on your keyboard takes too long. So, here are some keyboard shortcuts you can use to save you time and ultimately a headache:
```

Comment:
@CBID2 I hope you bear with me here and I thank you in advance for your changes 😄 . Would you mind addressing the last thing above so I can review the final version and merge? 

Basically line 165 should be changed into an intro for the shortcut usage. Less fun but more descriptive for learners: 
My proposal that you are free to tweak to be more proper English than mine. for example: 
>"For starting a comment or converting a line to a comment, you can use the following shortcut in VSCode in your programming files"

Reply:
Hi @ManonLef. Thanks for your suggestion. I noticed that the TOP lessons use a casual and humorous tone, so I think it would be best if the writing style in line 165 stays as is. I find that it makes the content more relatable and accessible, something that I don't see often in coding curricula.